bears/go/GoTOMLBear.py

```diff
@@ -0,0 +1,22 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a TOML file.
+    """
+    LANGUAGES = {'TOML'}
+    REQUIREMENTS = {GoRequirement('github.com/BurntSushi/toml/cmd/tomlv',
+                                  version='0.1.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ()
```

Comment:
This line is not covered by the tests, please add a test to make sure `create_arguments` outputs correctly.

https://travis-ci.org/coala/coala-bears/jobs/198859277#L3287